cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSIUtils.java

```diff
@@ -107,8 +107,17 @@ public static String encodeSTHCollection(String in) {
         } // if
         
         if (attrType.equals("geo:json")) {
-            return new ImmutablePair("ST_GeomFromGeoJSON('" + attrValue + "')", true);
+            return new ImmutablePair("ST_GeomFromGeoJSON(" + attrValue + ")", true);
         } // if
+
+        // TBD: What about:  ?
+          // 'geo:line'
```

Comment:
Also geo:point is usually used.

Reply:
It seems that geo:point is correctly covered, see https://github.com/telefonicaid/fiware-cygnus/tree/master/cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/utils/NGSIUtils.java#L97

Reply:
NTC (I guess)